Solutions/GoogleCloudPlatformIAM/Data Connectors/IAM_CCP/Iamtable.json

```diff
@@ -0,0 +1,377 @@
+{
+    "name": "GCP_IAMV2_CL",
+    "apiVersion": "2022-10-01-preview",
+    "type": "Microsoft.OperationalInsights/workspaces/tables",
+    "location": "{{location}}",
+    "properties": {
+        "schema": {
+            "name": "GCP_IAMV2_CL",
+            "columns": [
+                {
+                    "name": "TimeGenerated",
+                    "type": "datetime"
+                },
+                {
+                    "name": "receiveTimestamp",
+                    "type": "datetime"
+                },
+                {
+                    "name": "insertId",
+                    "type": "string"
+                },
+                {
+                    "name": "AuthInfoPrincipalEmail",
+                    "type": "string"
+                },
+                {
+                    "name": "AuthInfoPrincipalSubject",
+                    "type": "guid"
+                },
+                {
+                    "name": "plmethodname",
+                    "type": "string"
+                },
+                {
+                    "name": "resource_labels_project_id",
+                    "type": "string"
+                },
+                {
+                    "name": "pl_type",
+                    "type": "string"
+                },
+                {
+                    "name": "pl_requestMetadata_callerIp",
+                    "type": "string"
+                },
+                {
+                    "name": "pl_ServiceName",
+                    "type": "string"
+                },
+                {
+                    "name": "pl_requestMetadata_callerSuppliedUserAgent",
+                    "type": "string"
+                },
+                {
+                    "name": "pl_authorizationInfo",
```

Comment:
use pascal casing for naming convention of fields name

for example 

 resource_labels_version => ResourceLabelsVersion